sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -201,8 +201,8 @@ SUBROUTINE MDLFLD
       ALLOCATE(PBLRI  (IM,JSTA_2L:JEND_2U))    
 !     
 !     SECOND, STANDARD NGM SEA LEVEL PRESSURE.
+      CALL NGMSLP   ! this value is used in some later calculation.
```

Comment:
@EricJames-NOAA I understand you want to call this subroutine as variables computed in this subroutine are used later.
I'd like to propose if still put it in a if block so we don't have to call it if not needed.  For example,
If(IGET(105)>0  .and. IGET(xxx)>0)
  CALL NGMSLP

Reply:
@HuiyaChuang-NOAA this is a good suggestion, thanks.  I'm just doing a quick test on this change, and will push it soon.